scripts/earthengine/process_events.py

```diff
@@ -1034,29 +1136,239 @@ def read_active_events(self, filename: str):
         logging.info(
             f'Loaded {len(active_events)} events from file: {filename}')
 
+    def write_events_place_svobs(
+            self,
+            output_path: str,
+            event_ids: list = None,
+            event_props: list = ['area', 'count'],
+            date_formats: list = ['YYYY-MM-DD', 'YYYY-MM', 'YYYY']) -> list:
+        '''Generate CSV with SVObs for affected places of events.'''
+        if not event_ids:
+            event_ids = self.get_all_event_ids()
+        logging.info(
+            f'Generating place svobs for {len(event_ids)} events for dates: {date_formats}'
+        )
+
+        # Aggregation settings for event properties across places for a date.
+        property_config_per_date = {
+            'aggregate': 'sum',
+            'area': {
+                'aggregate': 'sum'
+            },
+            'EventId': {
+                'aggregate': 'set'
+            }
+        }
```

Comment:
nit: Can these constants be hoisted out to make the function logic more accessible?

Reply:
Done